### PR TITLE
Removed sdx-common clone in docker

### PR DESCRIPTION
**Pull request by emilioward**
This PR fixes the issue of a cryptography error when running docker build in sdx-receipt-rrm

## Checklist
  - [X] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by emilioward** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

File: Dockerfile [hunk 1]
```diff
@@ -6,8 +6,6 @@ COPY startup.sh /startup.sh
 RUN apt-get update -y
 RUN apt-get upgrade -y
 RUN apt-get install -yq git gcc make build-essential python3-dev python3-reportlab
-RUN git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
-RUN pip3 install ./sdx-common
```
Comment (necrophonic) [inline Dockerfile:11]: Docker container isn't starting:
```
/usr/bin/python3: Error while finding spec for 'app.consumer' (<class 'ImportError'>: No module named 'sdx')
```

Looks like the dockerfile needs to import the `requirements.txt` and run a `pip3` install on it

**Review by necrophonic — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (gemmai95) [reply Dockerfile:11]: This has now been addressed and tested

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

**Review by JamesGardiner — APPROVED** [review]

**Review by dcdarrell9 — APPROVED** [review]
LGTM 👍

**Review by necrophonic — APPROVED** [review]

**Pull request merged**